### PR TITLE
chore(costmodels): rearrange cost models files structure

### PR DESCRIPTION
**Pull request by boaz0**
- Move `costModelsDetails` and `createCostModelWizard` to `costModels`
- Move components that are used on both `createCostModelWizard` and `costModelsDetails` to `components`.

### CODE REVIEW

File: src/pages/costModels/costModelsDetails/components/addRateModal.test.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { fireEvent, render } from '@testing-library/react';
 import { createModel } from '@xstate/test';
-import { CostModel } from 'api/costModels';
-import { addRateMachine } from 'pages/createCostModelWizard/addPriceList';
 import React from 'react';
+import { CostModel } from 'api/costModels';
+import { addRateMachine } from '../../components/addPriceList';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/addRateModal.test.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/costModelsDetails/components/addRateModal.tsx [hunk 2]
```diff
@@ -31,7 +26,12 @@ import {
   SetMetric,
   SetRate,
   unusedRates,
-} from './addCostModelRateForm';
+} from '../../components/addCostModelRateForm';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/addRateModal.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/costModelsDetails/components/addRateModal.tsx [hunk 3]
```diff
@@ -31,7 +26,12 @@ import {
...
+import {
+  addRateMachine,
+  CurrentStateMachine,
+} from '../../components/addPriceList';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/addRateModal.tsx]: Please replace relative paths with absolute

File: src/pages/costModels/costModelsDetails/addSourceStep.tsx [hunk 4]
```diff
@@ -11,16 +11,16 @@ import { Provider } from 'api/providers';
...
-import { WarningIcon } from 'pages/createCostModelWizard/warningIcon';
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
 import { connect } from 'react-redux';
 import { createMapStateToProps } from 'store/common';
 import { sourcesActions, sourcesSelectors } from 'store/sourceSettings';
+import {
+  addMultiValueQuery,
+  removeMultiValueQuery,
+} from '../components/filterLogic';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/addSourceStep.tsx]: Please replace relative paths (i.e., ../components) with absolute

File: src/pages/costModels/costModelsDetails/components/costModelsDetailsToolbar.tsx [hunk 5]
```diff
@@ -19,15 +19,15 @@ import {
...
 import {
   addMultiValueQuery,
   addSingleValueQuery,
   removeMultiValueQuery,
   removeSingleValueQuery,
-} from 'pages/createCostModelWizard/filterLogic';
-import React from 'react';
-import { InjectedTranslateProps, translate } from 'react-i18next';
-import { Omit } from 'react-redux';
+} from '../../components/filterLogic';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/costModelsDetailsToolbar.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/costModelsDetails/components/priceListTable.tsx [hunk 6]
```diff
@@ -17,16 +17,16 @@ import { AxiosError } from 'axios';
...
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
 import { connect } from 'react-redux';
-import { createMapStateToProps } from 'store/common';
 import { FetchStatus } from 'store/common';
+import { createMapStateToProps } from 'store/common';
 import { costModelsActions, costModelsSelectors } from 'store/costModels';
 import { metricsSelectors } from 'store/metrics';
+import CostModelRateItem from '../../components/costModelRateItem';
+import { PriceListToolbar } from '../../components/Datatoolbar';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/priceListTable.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/costModelsDetails/components/table.tsx [hunk 7]
```diff
@@ -13,12 +13,12 @@ import { DollarSignIcon } from '@patternfly/react-icons';
...
 import { EmptyFilterState } from 'components/state/emptyFilterState/emptyFilterState';
+import React from 'react';
+import { InjectedTranslateProps, translate } from 'react-i18next';
 import {
   addMultiValueQuery,
   removeMultiValueQuery,
-} from 'pages/createCostModelWizard/filterLogic';
-import React from 'react';
-import { InjectedTranslateProps, translate } from 'react-i18next';
+} from '../../components/filterLogic';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/table.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/costModelsDetails/costModelInfo/header.tsx [hunk 8]
```diff
@@ -13,15 +13,15 @@ import {
...
-import UpdateCostModelDialog from 'pages/costModelsDetails/components/updateCostModel';
-import { styles } from 'pages/costModelsDetails/costModelsDetails.styles';
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
 import { connect } from 'react-redux';
 import { createMapStateToProps } from 'store/common';
 import { costModelsActions, costModelsSelectors } from 'store/costModels';
+import Dialog from '../components/dialog';
+import Dropdown from '../components/dropdown';
+import UpdateCostModelDialog from '../components/updateCostModel';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/costModelInfo/header.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/createCostModelWizard/context.ts [hunk 9]
```diff
@@ -1,5 +1,5 @@
-import { MetricHash } from 'api/metrics';
 import React from 'react';
+import { MetricHash } from '../../../api/metrics';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/context.ts]: Please replace relative paths (i.e., ../../../api) with absolute

File: src/pages/costModels/createCostModelWizard/priceList.tsx [hunk 10]
```diff
@@ -1,17 +1,11 @@
-import { Metric, MetricHash } from 'api/metrics';
+import { MetricHash } from 'api/metrics';
 import React from 'react';
 import { assign, interpret, Machine, State } from 'xstate';
-import AddPriceList from './addPriceList';
+import AddPriceList from '../components/addPriceList';
+import { TierData } from '../components/addPriceList';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/priceList.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/createCostModelWizard/priceListTable.tsx [hunk 11]
```diff
@@ -13,18 +13,18 @@ import {
...
-import CostModelRateItem from 'pages/costModelsDetails/components/costModelRateItem';
 import React from 'react';
 import { InjectedTranslateProps, Interpolate, translate } from 'react-i18next';
 import { connect } from 'react-redux';
-import { createMapStateToProps } from 'store/common';
 import { metricsSelectors } from 'store/metrics';
+import { createMapStateToProps } from '../../../store/common';
+import { TierData } from '../components/addPriceList';
+import CostModelRateItem from '../components/costModelRateItem';
+import { PriceListToolbar } from '../components/Datatoolbar';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/priceListTable.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/createCostModelWizard/priceListTable.tsx [hunk 12]
```diff
@@ -13,18 +13,18 @@ import {
...
+import { PlusCircleIcon } from '@patternfly/react-icons';
 import { MetricHash } from 'api/metrics';
 import { EmptyFilterState } from 'components/state/emptyFilterState/emptyFilterState';
-import CostModelRateItem from 'pages/costModelsDetails/components/costModelRateItem';
 import React from 'react';
 import { InjectedTranslateProps, Interpolate, translate } from 'react-i18next';
 import { connect } from 'react-redux';
-import { createMapStateToProps } from 'store/common';
 import { metricsSelectors } from 'store/metrics';
+import { createMapStateToProps } from '../../../store/common';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/priceListTable.tsx]: Please replace relative paths (i.e., ../../../store) with absolute

File: src/pages/costModels/createCostModelWizard/review.tsx [hunk 13]
```diff
@@ -19,14 +19,14 @@ import {
...
 import { MetricHash } from 'api/metrics';
-import CostModelRateItem from 'pages/costModelsDetails/components/costModelRateItem';
 import React from 'react';
 import { InjectedTranslateProps, Interpolate, translate } from 'react-i18next';
 import { connect } from 'react-redux';
-import { createMapStateToProps } from 'store/common';
 import { metricsSelectors } from 'store/metrics';
+import { createMapStateToProps } from '../../../store/common';
+import CostModelRateItem from '../components/costModelRateItem';
+import { WarningIcon } from '../components/warningIcon';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/review.tsx]: Please replace relative paths (i.e., ../../components) with absolute

File: src/pages/costModels/createCostModelWizard/review.tsx [hunk 14]
```diff
@@ -19,14 +19,14 @@ import {
 } from '@patternfly/react-core';
 import { OkIcon } from '@patternfly/react-icons';
 import { MetricHash } from 'api/metrics';
-import CostModelRateItem from 'pages/costModelsDetails/components/costModelRateItem';
 import React from 'react';
 import { InjectedTranslateProps, Interpolate, translate } from 'react-i18next';
 import { connect } from 'react-redux';
-import { createMapStateToProps } from 'store/common';
 import { metricsSelectors } from 'store/metrics';
+import { createMapStateToProps } from '../../../store/common';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/review.tsx]: Please replace relative paths (i.e., ../../../store) with absolute

**Comment by dlabrecq** [comment]
Is there an issue we can attach to this for QA testing?

**Comment by boaz0** [comment]
related #1380 1380

**Comment by dlabrecq** [comment]
Just need to resolve the merge conflict

File: src/pages/costModels/costModelsDetails/addSourceStep.tsx [hunk 15]
```diff
@@ -14,13 +14,13 @@ import { LoadingState } from 'components/state/loadingState/loadingState';
...
   removeMultiValueQuery,
-} from 'pages/createCostModelWizard/filterLogic';
-import { WarningIcon } from 'pages/createCostModelWizard/warningIcon';
+} from 'pages/costModels/components/filterLogic';
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
 import { connect } from 'react-redux';
 import { createMapStateToProps } from 'store/common';
 import { sourcesActions, sourcesSelectors } from 'store/sourceSettings';
+import { WarningIcon } from '../components/warningIcon';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/addSourceStep.tsx]: Looks like the relative paths were merged back in?

File: src/pages/costModels/costModelsDetails/components/updateRateModel.tsx [hunk 16]
```diff
@@ -20,7 +20,7 @@ import { MetricHash } from 'api/metrics';
 import { Form } from 'components/forms/form';
 import React from 'react';
 import { InjectedTranslateProps } from 'react-i18next';
-import { canSubmit, isRateValid } from './addCostModelRateForm';
+import { canSubmit, isRateValid } from '../../components/addCostModelRateForm';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/components/updateRateModel.tsx]: Please replace the relative paths (i.e., ../) with absolute

File: src/pages/costModels/costModelsDetails/costModelInfo/index.tsx [hunk 17]
```diff
@@ -1,11 +1,11 @@
...
 import { css } from '@patternfly/react-styles';
 import { CostModel, CostModelProvider } from 'api/costModels';
-import MarkupCard from 'pages/costModelsDetails/components/markup';
-import PriceListTable from 'pages/costModelsDetails/components/priceListTable';
-import SourceTable from 'pages/costModelsDetails/components/sourceTable';
-import { styles } from 'pages/costModelsDetails/costModelsDetails.styles';
 import React from 'react';
+import MarkupCard from '../components/markup';
+import PriceListTable from '../components/priceListTable';
+import SourceTable from '../sourceTable';
```
Comment (dlabrecq) [inline src/pages/costModels/costModelsDetails/costModelInfo/index.tsx]: Please replace the relative paths (i.e., ../) with absolute

File: src/pages/costModels/createCostModelWizard/table.tsx [hunk 18]
```diff
@@ -15,10 +15,13 @@ import { Table, TableBody, TableHeader } from '@patternfly/react-table';
 import { LoadingState } from 'components/state/loadingState/loadingState';
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
+import {
+  addMultiValueQuery,
+  removeMultiValueQuery,
+} from '../components/filterLogic';
+import { WarningIcon } from '../components/warningIcon';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/table.tsx]: Please replace the relative paths (i.e., ../) with absolute

File: src/pages/costModels/createCostModelWizard/table.tsx [hunk 19]
```diff
@@ -13,12 +13,15 @@ import {
 } from '@patternfly/react-core';
 import { Table, TableBody, TableHeader } from '@patternfly/react-table';
 import { LoadingState } from 'components/state/loadingState/loadingState';
+import { WarningIcon } from 'pages/costModels/components/warningIcon';
 import React from 'react';
 import { InjectedTranslateProps, translate } from 'react-i18next';
+import {
+  addMultiValueQuery,
+  removeMultiValueQuery,
+} from '../components/filterLogic';
```
Comment (dlabrecq) [inline src/pages/costModels/createCostModelWizard/table.tsx]: one more

**Pull request merged**